plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -179,16 +214,14 @@ def eval_deisolate(self, port_name):
 
         # we need some time after the change in state
         elif datetime.now() >= self.ports_states[port_name].get_change_time() + timedelta(minutes=self.deisolate_consider_time):
-            port_data = self.ports_data.get(port_name)
-            port_speed = port_data.get(Constants.ACTIVE_SPEED)
-            port_asic = port_data.get(Constants.ASIC)
-            port_width = port_data.get(Constants.WIDTH)
-            fec_mode = port_data.get(Constants.FEC_MODE)
-            raw_ber_rate, eff_ber_rate, symbol_ber_rate = self.calc_ber_rates(port_name, port_speed, port_width)
-            if self.check_ber_threshold(port_name, port_speed, port_asic, fec_mode, raw_ber_rate, eff_ber_rate, symbol_ber_rate, isolation=False):
-                cause = Constants.ISSUE_BER
-                self.ports_states[port_name].update(Constants.STATE_ISOLATED, cause)
-                return
+            port_obj = self.ports_data.get(port_name)
+            if port_state.cause == Constants.ISSUE_BER:
```

Comment:
what is port_state.cause? where did you get port_state? may be port_obj?

Reply:
port cause is an attribute of a PortState struct of a port that got isolated